multimedia/ffmpeg/Makefile

```diff
@@ -345,7 +345,7 @@ endef
 define Package/libffmpeg-full
 $(call Package/libffmpeg/Default)
  TITLE+= (full)
- DEPENDS+= +alsa-lib +PACKAGE_libopus:libopus \
+ DEPENDS+= +alsa-lib +libgnutls +PACKAGE_libopus:libopus \
```

Comment:
Does ffmpeg actually support mbedtls?

Reply:
Yes, it does

Reply:
I would think it's better to use that. It's smaller.

Reply:
hrm OK.